yarn.lock

```diff
@@ -0,0 +1,10690 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Don't use `yarn`. Use `npm`

Reply:
@larsbs I used both for various projects and yarn seems to be better in a way, that it is caching a lot of libraries, which makes it work faster. Do you have any information that might be interesting for comparison? At company we are using npm?

Reply:
Yes, at company we use `npm`. And there are various benefits for using `npm` over `yarn` (starting from version 5, that's the important thing):

 - Independent tool (yarn is basically backed by fb) 
 - Comes with node and you can expect to find it in every other system
 - The newer versions of `npm` also include a lock file basically removing the biggest different in favor of `yarn`
 - Better support for compiling native libraries (this was the main reason that made us move away)
 - Using `npm ci` intead of `npm i`, the speed difference between `yarn` and `npm` is basically none now.
 - And a few more I don't want to continue listing like checking dependencies for vulnerabilities, easier upgrade of dependencies, etc.

`yarn` used to be "better" than `npm` but not anymore, so the main gist is, why learn another tool when the default one already does everything?

I hope that helped you

Reply:
@larsbs Thank you! This is really valuable knowledge! I will read more about changes of the npm itself, but as I have a lot of tasks now, it will be added to my personal backlog :-)